src/Statement.php

```diff
@@ -100,12 +90,9 @@ public function executeStatement(array $params = []): int
      */
     private function executeWithRetry(callable $callable, ...$params)
     {
-        $parentCall = \Closure::fromCallable($callable);
-        $parentCall->bindTo($this, parent::class);
-
         try {
             attempt:
-            $result = $parentCall(...$params);
+            $result = $callable(...$params);
```

Comment:
below mutation: never worked with mutation testing ever, so unsure how to handle this...

Reply:
This mutation error was present in the previous version too. I already improved the MSI thanks to #86, I think I'll introduce https://packagist.org/packages/roave/infection-static-analysis-plugin in this branch to improve it further.